pymatgen/vis/structure_chemview.py

```diff
@@ -15,9 +14,9 @@
     chemview_loaded = True
 except ImportError:
     chemview_loaded = False
+    MolecularViewer = get_atom_color = None
 
 
-@requires(chemview_loaded, "To use quick_view, you need to have chemview installed.")
```

Comment:
@janosh. Please give me some advice on this. If we want to honor `pyright`, `requires` should be dropped then?